src/ast/mod.rs

```diff
@@ -61,6 +63,46 @@ mod value;
 #[cfg(feature = "visitor")]
 mod visitor;
 
+pub trait Convert<T>
+where
+    Self: Into<T>,
+{
+    #[inline(always)]
```

Comment:
I doubt if this hint works on trait definition

Reply:
According to my investigation, the `#[inline(always)]` will affect trait method with function body，but it is highly possible the compiler will automatically inline it even without `#[inline(always)]` due to there is no dynamic call